disasters/hurricane-harvey/0831/Houston-East-20170831-103f-100d-0f4f-RGB.json

```diff
@@ -36,17 +36,15 @@
   },
 
   "links": [
-    {
-      "rel": "self",
-      "href": "https://storage.googleapis.com/pdd-stac/disasters/hurricane-harvey/0831/Houston-East-20170831-103f-100d-0f4f-RGB.json"
-    },
     {
       "rel": "collection",
-      "href": "../../catalog.json"
+      "href": "../../collections/planet-collection.json",
+      "href:from": "../hurricane-harvey/0831/Houston-East-20170831-103f-100d-0f4f-RGB.json"
```

Comment:
@mojodna this should be hurricane-harvey/0831/..., right?

Reply:
No, this is right. From `collections/`, the reverse link needs to go up a level to descend a different sibling.